src/faction.cpp

```diff
@@ -99,6 +101,13 @@ faction_template::faction_template( const JsonObject &jsobj )
     lone_wolf_faction = jsobj.get_bool( "lone_wolf_faction", false );
     load_relations( jsobj );
     mon_faction = jsobj.get_string( "mon_faction", "human" );
+    for( const JsonObject &jao : jsobj.get_array( "epilogues" ) ) {
+        epilogue_data.emplace( std::make_tuple<int, int, std::string>(
```

Comment:
If you call any `emplace` (or `emplace_back`) function, you can supply the constructor arguments directly to that function. In other words: instead of `foo.insert( the_element_type( a, b, c ) )`, you can write `foo.emplace( a, b, c )`.

In your code, you construct the tuple (via `std::make_tuple`) and also add a copy of it via `emplace`.

Suggestion: remove the call to `std::make_tuple` and forward the values directly to `epilogue_data.emplace`.